tests/utils/dev-server.js

```diff
@@ -37,10 +37,13 @@ const startServer = async ({ cwd, env = {}, args = [] }) => {
     cwd,
     env: { BROWSER: 'none', ...env },
   })
-  return new Promise((resolve, reject) => {
+  let output = ''
+  const serverPromise = new Promise((resolve, reject) => {
     let selfKilled = false
-    ps.stdout.on('data', (data) => {
-      if (data.toString().includes('Server now ready on')) {
+    ps.stdout.on('data', (dataBuffer) => {
+      const data = dataBuffer.toString()
```

Comment:
I believe using the `encoding: 'utf-8'` option of `execa` should provide with `data` string chunks instead directly. Would this work?

Reply:
Sure, done in https://github.com/netlify/cli/pull/1694/commits/a0d81587763b7673c23de5b294f02cb7a873b63b